src/coreclr/src/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -481,8 +477,31 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                                                     INS_OPTS_NONE);
                     }
                 }
-            }
-            break;
+                break;
+            case NI_Vector64_Create:
+            case NI_Vector128_Create:
+                if (intrin.op1->isContainedFltOrDblImmed())
+                {
+                    const double dataValue = intrin.op1->AsDblCon()->gtDconVal;
+                    GetEmitter()->emitIns_R_F(INS_fmov, emitSize, targetReg, dataValue, opt);
+                }
+                else if (varTypeIsFloating(intrin.baseType))
+                {
+                    GetEmitter()->emitIns_R_R_I(ins, emitSize, targetReg, op1Reg, 0, opt);
+                }
+                else
+                {
+                    if (intrin.op1->isContainedIntOrIImmed())
+                    {
+                        const ssize_t dataValue = intrin.op1->AsIntCon()->gtIconVal;
+                        GetEmitter()->emitIns_R_I(INS_movi, emitSize, targetReg, dataValue, opt);
```

Comment:
Ah I see that `INS_movi` will try the two's compliment and `mvni` as well which is good!